pkg/services/x509/uzi_validator.go

```diff
@@ -0,0 +1,151 @@
+package x509
+
+import (
+	"crypto/x509"
+	"fmt"
+	"strings"
+
+	"github.com/lestrrat-go/jwx/jwa"
+
+	"github.com/nuts-foundation/nuts-auth/assets"
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+	"github.com/nuts-foundation/nuts-auth/pkg/services"
+)
+
+// Uzi Attributes used to sign the message
+// See table 12 on page 62 of the Certification Practice Statement (CPS) UZI-register v10.x
+// https://zorgcsp.nl/Media/Default/documenten/2020-05-06_RK1%20CPS%20UZI-register%20V10.0.pdf
+var UziAttributes = []string{
```

Comment:
Since slices/arrays are passed by reference and thus mutable, is `UziAttributes` intended to be modified at runtime? Especially since it's exported.

If it's intended to be immutable I'd make an exported function that returns the (new) array. Is also a bit more flexible when we want to change the implementation later on (load from bin/disk for instance).